src/app/models/Client.ts

```diff
@@ -5,7 +5,7 @@ export interface Client {
   phone: string;
   email: string;
   address: Address;
-  id:Number;
+  id: Number;
```

Comment:
Hey @MBenedicte, did we mean to write `number` or we have a Number` custom type as it's not built into typescript?.

Reply:
worked on that @CryceTruly